src/gui/mzroll/mainwindow.h

```diff
@@ -100,6 +101,7 @@ class LogWidget;
 class SpectralHit;
 class SpectralHitsDockWidget;
 class PeptideFragmentationWidget;
+class Notificator;
```

Comment:
Well, either we need to include the file or we forward declare it. Both of them are not required
Since we do not require any info related to Notificator here, just including it in the cpp file will do the job